### PR TITLE
[book] document example of manual arbitrary impl

### PR DESCRIPTION
**Pull request by matthew-russo**
replaces #282

### CODE REVIEW

File: book/src/proptest/tutorial/arbitrary.md [hunk 1]
```diff
@@ -10,5 +10,86 @@ In some cases, where it makes sense to define a canonical strategy, such as in
...
+## Deriving `Arbitrary`
+
 The experimental [`proptest-derive` crate](../../proptest-derive/index.md) can
-be used to automate implementing `Arbitrary` in common cases.
+be used to automate implementing `Arbitrary` in common cases. For example, imagine we have a struct that represents a point in a 2-D coordinate space:
+```rust
+#[derive(Debug)]
+struct Point {
+  x: i32,
+  y: i32,
```
Comment (rexmas) [inline book/src/proptest/tutorial/arbitrary.md:22]: 4 spaces instead of 2

**Review by rexmas — APPROVED** [review]
small change otherwise looks good to me :)

File: book/src/proptest/tutorial/arbitrary.md (hunk 1 above)
Comment (matthew-russo) [reply book/src/proptest/tutorial/arbitrary.md]: fixed

**Comment by matthew-russo** [comment]
i can't reproduce the build failure and this is just docs changes, so i'm going to merge

**Pull request merged**